services/movie.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids is not None:
+        movies = movies.filter(genres__id__in=genres_ids)
+
+    if actors_ids is not None:
+        movies = movies.filter(actors__id__in=actors_ids)
+
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:

```

Comment:
thsi function returns only one movie, not a queryset

Reply:
@Mykhailo-Benzel Not fixed